docgen/src/code_description.py

```diff
@@ -53,8 +79,11 @@ def describe_file_classes(llm, filepath):
 
 def describe_classes(llm, parsed_code):
     (src, tree) = parsed_code
-    func_llm = llm.with_structured_output(ClassDescription)
-    return [func_llm.invoke(func) for func in get_definitions(src, tree, definition_type="class_definition")]
+    system = class_struct_example
```

Comment:
LangChain has some built-in support for few-shot prompting that could be better: https://python.langchain.com/docs/how_to/few_shot_examples/